src/Codecool.FilePartReader.UnitTests/FilePartReaderTests.cs

```diff
@@ -79,6 +83,7 @@ public void ReadLines_CreateString_ReturnResult()
         [Test]
         public void GetWordsOrderedAlphabetically_SortWords_ReturnSortedList()
```

Comment:
This is a `FileWorldAnalyzer` test not `FilePartReaderTest`. It should be in a separated file.